src/main/java/com/project/bookapp/model/Book.java

```diff
@@ -0,0 +1,34 @@
+package com.project.bookapp.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
don't use `@Data` for entities

```suggestion
@Getter
@Setter
@Table(name....
```